src/pytest/test.py

```diff
@@ -416,9 +416,14 @@ def testCustomStopwords(env):
     # Index with custom stopwords
     env.assertOk(r.execute_command('ft.create', 'idx2', 'stopwords', 2, 'hello', 'world',
                                     'schema', 'foo', 'text'))
+    res = env.cmd('ft.info', 'idx2')
+    env.assertEqual(res[39], ['hello', 'world'])
```

Comment:
@ashtul there is no title for this info?

Reply:
@gkorland 
The title is in no. 38. `stop_word_list`